test/ctia/logger_test.clj

```diff
@@ -0,0 +1,36 @@
+(ns ctia.logger-test
+  (:require [ctia.logging :refer :all]
+            [ctia.events :as c :refer :all]
+            [clojure.test :as t :refer :all]
```

Comment:
In general, try to avoid :refer :all style requirements.  _Maybe_ it is okay for clojure.test, but I would argue against doing a :refer :all for the system under test.  The (init!) function below is a common function name, so I can't tell where it is coming from.